src/components/Icon/Icon.tsx

```diff
@@ -159,41 +159,44 @@ export const Icon = React.forwardRef<SVGSVGElement, IconProps>(
 			}
 		}, [border, safeRef]);
 
+		const classname = classnames('tc-icon', className, transform, {
+			[`tc-icon-name-${name}`]: !(isImg || isRemote),
+		});
+
 		if (isImg) {
 			return (
-				<img alt="" src={name.substring(4)} className="tc-icon" {...accessibility} {...rest} />
+				<img
+					alt=""
+					src={name.substring(4)}
+					className={classnames('tc-icon', classname)}
```

Comment:
```suggestion
					className={classname}
```

`tc-icon` is already in `classname`